sites/bizbash.com/server/templates/website-section/venues-destinations.marko

```diff
@@ -76,9 +76,18 @@ $ const { id, alias, name, pageNode } = data;
               name="website-scheduled-content"
               params={ sectionAlias: child.alias, limit: 4, queryFragment }
             >
-              <div class="node-list__header">
-                <marko-web-link href=`/${child.alias}`>${child.name}</marko-web-link>
-              </div>
+                <if (child.name === 'North America' || child.name === 'Global')>
```

Comment:
For complex checks, we prefer using [Array.includes()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) to reduce line length and increase legibility.

Something like `<if (['North America', 'Global'].includes(child.name))>`. If the number of values needs to increase it can be defined as a variable separately and checked inline.